R/new_factory.R

```diff
@@ -4,86 +4,132 @@
 #' factory is created with a template of report, and the working environment is
 #' moved to the newly created factory.
 #'
-#' @details
-#' The default factory includes:
-#'
-#' \itemize{
-#'
-#'  \item \code{data/}: a folder storing data
-#'
-#'  \item \code{report_sources/}: a folder storing the reports, named after the
-#' convention described in \code{\link{update_reports}}, and stored in
-#' subfolders \code{contacts} and \code{epicurves}
-#'
-#'  \item \code{.gitignore}: a file used to tell git to ignore the produced
-#' outputs in \code{report_outputs}
-#'
-#'  \item \code{open.Rproj}: an Rproject to open the factory using Rstudio
+#' @param factory The name of the report factory to be created.
+#' @param path The folder where the report factory should be created.  This
+#'   will default to the current directory.
+#' @param report_sources The name of the folder to be used for report
+#'   templates.  Defaults to 'report_sources/'.
+#' @param outputs The name of the folder to be used for saving the built
+#'   reports.  Defaults to 'outputs/'.
+#' @param move_in A `logical` indicating if the current session should move into
+#'   the created factory; defaults to `TRUE`.
+#' @param create_README A `logical` indicating if a 'README' file should be
+#'   created; defaults to TRUE.
+#' @param create_example_report A `logical` indicating if `new_factory()` should
+#'   create an example report in the 'report_sources' folder along with some
+#'   example data in the 'data/raw' folder; defaults to TRUE.
+#' @param create_data_folders a `logical` indicating if `new_factory()` should
+#'   create folders to store data; defaults to TRUE.
+#' @param create_scripts_folder a `logical` indicating if `new_factory()` should
+#'   create folders to store R scripts; defaults to TRUE.
+#' @param use_here a `logical` indicating if `new_factory()` should create
+#'   a `.here` that can be used with `here::here()`.
```

Comment:
Add: 
Defaults to `TRUE`

Reply:
closed with https://github.com/reconhub/reportfactory/pull/68/commits/e60198f05faaf899a33aab96f3d9634ab3413252